.github/workflows/ci.yml

```
@@ -0,0 +1,85 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
+
+# This workflow is about project code.
+# It will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: CI
+
+on:
+  push:
+    branches: ["main"]
+    paths:
+      - ".github"
+      - "packages/**"
+      - "package.json"
+      - "package-lock.json"
+  pull_request:
+    branches: ["main"]
+    paths:
+      - ".github"
+      - "packages/**"
+      - "package.json"
+      - "package-lock.json"
+    types:
+      - opened
+      - ready_for_review
+      - reopened
+      - synchronize
```

```suggestion
on:
  push:
    branches: [ "main" ]
  pull_request:
    branches: [ "main"]
```

This will run CI on each commit of pr and for each main commit

---

Why wouldn't I want that?

---

i didnt get tour message

---

What would be bad about running CI whenever a PR is updated or when a new commit is pushed to `main`?

---

I think I misspoke, my proposed change allows run le ci on all commits including rebase, force push ...

---

Paths are important though, because i don't care about changes to other locations

---

Oh okay 👌